package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "webpack --mode production"
+    "serve": "webpack serve --mode development"
```

Comment:
[OPTIONAL]
- [ ] What do you think about changing the `"serve": "webpack serve --mode development` to `"start": "webpack serve --mode development"` or `"dev": "webpack serve --mode development"` ? That will help us follow the standard by running commands like: `npm run start` or `npm run dev` to start the server.